### PR TITLE
Dogepedia - Update how-can-my-business-accept-dogecoin.md

### PR DESCRIPTION
**Pull request by MishaBoar**
Typos and added more information about ecommerce solutions, with mentions to libdogecoin and DogeGarden.

### CODE REVIEW

File: content/dogepedia/articles/how-can-my-business-accept-dogecoin.md [hunk 1]
```diff
@@ -13,8 +13,10 @@ At the most foundational level, for your business to accept Dogecoin in exchange
 
 If you have a bricks-and-mortar business where payments are made in person, it’s common for merchants to print a QR code that is scanned by customers’ smartphones to streamline the transaction process.  
 
-If you have an ecommerce store, companies like Coinbase Commerce and BitPay make accepting Dogecoin a breeze, and include integrations with popular platforms like Shopify, WooCommerce, and Opencard!
+If you have an ecommerce store, companies like Coinbase Commerce and BitPay make accepting Dogecoin a breeze, and include integrations with popular platforms like Shopify, WooCommerce, and OpenCart. It is also possible to develop custom solutions connecting directly to a [Dogecoin Core node](/dogepedia/how-tos/operating-a-node/) through the JSON-RPC interface. The Dogecoin Foundation is currently developing [Libdogecoin](https://foundation.dogecoin.com/trailmap/libdogecoin/), a library that should, among other things, facilitate and standardize the implementation of Dogecoin payments.
```
Comment (RichieJohn) [inline content/dogepedia/articles/how-can-my-business-accept-dogecoin.md:16]: Perhaps change the word "should" to the word "will"?

**Review by RichieJohn — COMMENTED** [review]
Looks great. Only a minor suggestion - feel free to adopt or disregard. All good either way. :)

File: content/dogepedia/articles/how-can-my-business-accept-dogecoin.md (hunk 1 above)
Comment (MishaBoar) [reply content/dogepedia/articles/how-can-my-business-accept-dogecoin.md]: Changed!

**Review by xanimo — APPROVED** [review]
utACK

**Pull request merged**